base.js

```diff
@@ -0,0 +1,39 @@
+import Attribute from './attribute';
+
+class BaseModel {
```

Comment:
what would be nice is:
the proxy are set directly on the model so we can do myInstance.name = "something", this way the standard of model are respected. furthermore this value is set on attributes (plain object). This way we could print all values of attributes. At this moment we need to use the get to know the value when debugging. myInstance.attributes.name.get() (impossible to know by outputting the name directly).

Reply:
set by a proxy, this way we still can trigger validations and callbacks